manifests/namespace-install/argo-server-rbac/argo-server-rolebinding.yaml

```diff
@@ -9,3 +9,16 @@ roleRef:
 subjects:
 - kind: ServiceAccount
   name: argo-server
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: argo-cluster-wftmpl-binding
```

Comment:
I think you should either (a) settle on one single abbreviation and use it everywhere or (b) always use the full name